packages/components/src/components/entities/actions.ts

```diff
@@ -362,3 +377,43 @@ export function deleteEntityType(deleteActionName: string, rowId: number): Promi
         });
     });
 }
+
+export function parentValuesDiffer(sortedOriginalParents: List<EntityChoice>, currentParents: List<EntityChoice>) : boolean {
+    const sortedCurrentParents = currentParents.sortBy(choice => choice.type ? choice.type.label : "~~NO_TYPE~~", naturalSort).toList();
+    const difference = sortedOriginalParents.find((original, index) => {
+        const current = sortedCurrentParents.get(index);
+        if (!current)
+            return true;
+        if (current.type && original.type.rowId !== current.type.rowId) {
+            return true;
+        }
+        const originalValues = original.value ? original.value.split(DELIMITER).sort(naturalSort).join(DELIMITER) : "";
+        const currentValues = current.value ? current.value.split(DELIMITER).sort(naturalSort).join(DELIMITER) : "";
```

Comment:
This part about split on delimiter -> sort -> join on delimiter isn't a lot of duplicate code, but I could see pulling that out to a little helper function so that it can have a quick jest test. Your call.

Reply:
Doesn't seem worth separate unit tests for usage of standard methods, so I'll leave it as is.